op-service/sources/reth_db_stub.go

```diff
@@ -0,0 +1,13 @@
+//go:build !rethdb
+
+package sources
+
+import (
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/core/types"
+)
+
+// FetchRethReceipts stub; Not available without `rethdb` build tag.
+func FetchRethReceipts(dbPath string, blockHash *common.Hash) (types.Receipts, error) {
+	panic("unimplemented! Did you forget to enable the `rethdb` build tag?")
```

Comment:
When / where do we set the build tag?

Reply:
When compiling the `op-node`, the `rethdb` build tag can be set like so:

`env GO111MODULE=on GOOS=$(TARGETOS) GOARCH=$(TARGETARCH) go build -tags rethdb -v $(LDFLAGS) -o ./bin/op-node ./cmd/main.go`